src/server/server_family.cc

```diff
@@ -783,11 +781,14 @@ static void RunStage(bool new_version, std::function<void(unsigned)> cb) {
   }
 };
 
+using PartialSaveOpts =
```

Comment:
I do not like tuples :) they are unreadable and then using cref to pass into reference becomes too complicated to understand. and it's really unnecessary because with 3 more lines you can define a dedicated struct that is more readable and simple.

Reply:
Me neither, but for such simple cases tuples are easily constructed and unpacked with structured bindings